docs/cache-provider.mdx

```diff
@@ -14,7 +14,7 @@ import { prefixer } from 'stylis'
 const myCache = createCache({
   key: 'my-prefix-key',
   stylisPlugins: [
-    customPlugin,
+    // customPlugin,
```

Comment:
Instead of removing this here, we can declare this before calling `createCache`:
```js
const customPlugin = () => {}
```